src/VTMap.js

```diff
@@ -0,0 +1,138 @@
+      zoom: 7,
+      style: { height: "600px", width: "600px" },
+      zoomControl: false,
+      scrollWheelZoom: false,
+      attributionControl: false,
+      disabled: true,
+    };
+  }
+  /*Click Handler for Game Start*/
+  handleClick = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      disabled: false,
+      gameStarted: true,
+      zoom: 18,
+    });
+  };
+/* Quit Button Handler*/
+  handleQuitButton = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      gameStarted: false,
+      zoom: 7,
+    });
+  };
+ /* Fetches Coordinates for Guess*/
+  getCurrentLoc = (event) => {
+    let zoomCoords = randomCoords();
+    this.handleClick(event);
+    let lat = zoomCoords[0];
+    let long = zoomCoords[1];
+    let request = `https://nominatim.openstreetmap.org/reverse?lat=${lat}&lon=${long}&format=geojson`;
+    this.setState({
+      center: zoomCoords,
+    });
+
+    return fetch(request)
+      .then((response) => response.json())
+      .then((response) => {
+        console.log(response);
+        let locInfo = response.features[0].properties.address;
+        let currentTown =
+          locInfo.town || locInfo.city || locInfo.village || locInfo.hamlet;
+        let currentCounty = locInfo.county;
+        this.setState({
+          town: currentTown,
+          county: currentCounty,
+        });
+      });
+  };
+ 
+  /* Renders Map and Handles Zoom Functions*/
+  render() {
+    let VTBorder = borderData.geometry.coordinates[0].map((coordSet) => {
+      return [coordSet[1], coordSet[0]];
+    });
+    console.log(VTBorder);
+
+    return (
+      <>
+        <div>
+          <Map
+            center={this.state.center}
+            zoom={this.state.zoom}
+            style={this.state.style}
+            zoomControl={this.state.zoomControl}
+            scrollWheelZoom={this.state.scrollWheelZoom}
+            attributionControl={this.state.attributionControl}
+          >
+            <TileLayer
+              url="https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}"
+              attribution="Tiles &copy; Esri &mdash; Source: Esri, i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community"
+            />
+            <Polygon positions={VTBorder} />
+          </Map>
+        </div>
+        <div>
+          <Buttons
+            disabled={this.state.disabled}
+            handleClick={this.getCurrentLoc}
+            guessButton={this.state.guessButton}
+            quitButton={this.state.quitButton}
+            randomLat={this.state.center[0]}
+            randomLong={this.state.center[1]}
+            currentTown={this.state.town}
+            currentCounty={this.state.county}
+          />
+        </div>
+        <div>
+          <Navigation
+            disabled={this.state.disabled}
+            handleClick={this.handleClick}
+            handleNorthClick={this.handleNorthClick}
```

Comment:
You are passing this as a prop, but you aren't using it on the component. Instead you're using a method that lives on the child component, so nothing happens when you click "Move North"